packages/clay-multi-select/src/ClayMultiSelect.js

```diff
@@ -92,6 +92,24 @@ class ClayMultiSelect extends ClayComponent {
 		}
 	}
 
+	/**
+	 * Handles the input blur
+	 * @protected
+	 */
+	_handleInputOnBlur() {
+		this._removeFocusedItem();
+		this._inputFocus = false;
+	}
+
+	/**
+	 * Handles the input focus
+	 * @protected
+	 */
+	_handleInputOnFocus() {
+		this._removeFocusedItem();
```

Comment:
Why do we need to do this here?

Reply:
Yes, this is not necessary here, we are already removing it in `_handleItemFocus`. Thanks!